integrations/purchase/bridge-api-categories.js

```diff
@@ -202,7 +202,8 @@ const idToCategory = {
 };
 
 export const BRIDGE_API_CATEGORIES = Object.fromEntries(
-  Object.entries(idToCategory).map(([id, purchaseType]) =>
-    purchaseType ? { purchaseType, activityType: ACTIVITY_TYPE_PURCHASE } : purchaseType
-  )
+  Object.entries(idToCategory).map(([id, purchaseType]) => [
```

Comment:
This is rather hard to read. 

If `{1: 'x', 2: null}` should map to `{1: {activityType: ..., purchaseType: ... }, 2: null}` then this should work:

```javascript
Object.entries(idToCategory).reduce((accumulator, [id, purchaseType]) => {
   accumulator[id] = purchaseType ? { activityType: ACTIVITY_TYPE_PURCHASE, purchaseType } : null;
   return accumulator;
}, {});
```

Reply:
Seems to work indeed and solves the compatibility issue 👍